src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -5,7 +5,6 @@ import { transformForSubmit } from 'us-forms-system/lib/js/helpers';
 const searchInstitutionBaseUrl = `${environment.API_URL}/v0/gi/institutions/search`;
 
 export function fetchInstitutions({ institutionQuery, page }) {
-  // const fetchUrl = institutionQuery ? `${searchInstitutionBaseUrl}?name=${institutionQuery}` : url;
```

Comment:
Addresses [this](https://github.com/department-of-veterans-affairs/vets-website/pull/8062#discussion_r206720188).